db/db_impl.cc

```diff
@@ -497,6 +500,8 @@ DBImpl::~DBImpl() {
     env_->UnlockFile(db_lock_);
   }
 
+  Log(InfoLogLevel::INFO_LEVEL, immutable_db_options_.info_log,
```

Comment:
Without this there's no way to tell from LOG whether the DB was destroyed cleanly or the process crashed. Or is there?